sass/components/_sponsors.scss

```diff
@@ -44,6 +44,7 @@
         ('tier': 'platinum', 'color': $color-white, 'height': 100px),
         ('tier': 'silver', 'color': #bbb, 'height': 80px),
         ('tier': 'bronze', 'color': #9f7d6c, 'height': 60px),
+        ('tier': 'infrastructure', 'color': $color-white, 'height': 60px),
```

Comment:
What do you think making the color a darker grey? "Iron grey". ^_^

```suggestion
        ('tier': 'infrastructure', 'color': #777, 'height': 60px),
```

Reply:
This is how it looks like:
![image](https://user-images.githubusercontent.com/188612/161908701-2f64753a-2444-4f67-b33a-8579ec5cf691.png)